crates/ruma-events/src/room/message.rs

```diff
@@ -35,6 +35,8 @@ mod text;
 mod video;
 
 pub use audio::{AudioInfo, AudioMessageEventContent};
+#[cfg(feature = "unstable-msc3245-v1-compat")]
+pub use audio::{UnstableAudioDetailsContentBlock, UnstableVoiceContentBlock};
```

Comment:
I'm wondering, do we need to re-export `Amplitude` too?

Reply:
It should be public elsewhere..? I'll check the docs preview once it's ready.

Reply:
Right so actually the issue seems to be that the `unstable-msc3245-v1-compat` cargo feature should also activate `unstable-msc3246`.